contracts/contracts/CalldataEncryption.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.0;
+
+import {Subcall} from "./Subcall.sol";
+import {Sapphire} from "./Sapphire.sol";
+import {CBOR_encodeBytes, CBOR_encodeUint} from "./CBOR.sol";
```

Comment:
```suggestion
import "./CBOR.sol" as CBOR;
```
according to the [announcement of free functions](https://soliditylang.org/blog/2020/09/02/solidity-0.7.1-release-announcement/), "it is recommended to use the import statement that generates a new object, so that the file's scope is not polluted needlessly"

This would also come with prefixing all calls to the free functions with CBOR.
Which then probably makes prefixing the functions in CBOR.sol obsolete.